worblehat-web/src/main/java/de/codecentric/psd/worblehat/web/formdata/InsertBookFormData.java

```diff
@@ -70,11 +72,23 @@ public void setEdition(String edition) {
         this.edition = edition;
     }
 
+    public String getDescription() {
+        return description;
+    }
+
+    public void setDescription(final String description) {
+        this.description = description;
+    }
+
     @Override
     public String toString() {
-        return "InsertBookFormData [title=" + title + ", edition=" + edition
-                + ", yearOfPublication=" + yearOfPublication + ", isbn=" + isbn + ", author=" + author
-                + "]";
+        return "InsertBookFormData{" +
+                "title='" + title + '\'' +
+                ", edition='" + edition + '\'' +
+                ", yearOfPublication='" + yearOfPublication + '\'' +
+                ", isbn='" + isbn + '\'' +
+                ", author='" + author + '\'' +
+                ", description='" + description + '\'' +
+                '}';
```

Comment:
Wir haben verändert, wie `toString` das Ergebnis ausgibt. Vorher war es in eckigen Klammern, jetzt haben wir geschweifte Klammern.

Zeit für Lombok!